frontend/src/metabase/visualizations/components/settings/ChartSettingFieldPicker.jsx

```diff
@@ -34,10 +37,11 @@ const ChartSettingFieldPicker = ({
         placeholder={t`Select a field`}
         placeholderNoOptions={t`No valid fields`}
         isInitiallyOpen={value === undefined}
+        hiddenIcons
```

Comment:
Not sure what this actually does. I tried removing it but don't see any difference. Am I missing something?

Reply:
This removes the checkmark next to the selected value in the drop down. The new designs don't have it anymore.
![image](https://user-images.githubusercontent.com/1328979/187212586-689d44fe-5022-4c8a-907f-11fff8c3c045.png)